lib/elastomer/version_support.rb

```diff
@@ -13,7 +13,7 @@ class VersionSupport
       parent
       refresh
     ].freeze
-    ES_2_X_INDEXING_PARAMETER_NAMES = %i[consistency ttl timestamp].freeze
+    ES_2_X_INDEXING_PARAMETER_NAMES = %i[type consistency ttl timestamp].freeze
```

Comment:
Is this supposed to be `op_type` or is it an unrelated change?

Reply:
oops thats a typo from a more invasive approach I rejected, will remove thanks!!